### ISSUE TITLE
Not saving labels and unable to create training set

### ISSUE THREAD

**Issue by WrathUM** [DeepLabCut/DeepLabCut#889]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). Please fill out the template completely, including the full "traceback" and input code that you ran to hit this error.

OS: Windows 10
DeepLabCut Version:  2.2
Anaconda env used: GPU

I used the GUI to label the videos and I can see the excel spreadsheets with values stored in the excel, but when I click the "Check Labels!" button in the GUI, it says that there are not any labeled data. 

![image](https://user-images.githubusercontent.com/57931859/92188591-db572e80-ee11-11ea-8273-6a33296af0c7.png)

Please place code inside this:

note to user: currently only ResNet50-v1 is available for maDLC
note to user: currently model comparison is not available in maDLC
Creating images with labels by Walter.
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_1_top does not appear to have labeled data!
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_2_top does not appear to have labeled data!
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_3_top does not appear to have labeled data!
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_1_side does not appear to have labeled data!
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_2_side does not appear to have labeled data!
Attention: C:\Users\FIP_computer\Trial Run -Walter-2020-08-26\labeled-data\test_video_3_side does not appear to have labeled data!
If all the labels are ok, then use the function 'create_training_dataset' to create the training dataset!

**How to Reproduce the problem**
1. Go to Deeplabcut 2.2 and load a previously labeled project
2. Click on label frames
3. Click on "Check Labels!"

This problem is likely due to some prior set-up or procedural problem that I am unaware about.

**Screenshots**
This is for the additional context section.
![image](https://user-images.githubusercontent.com/57931859/92188904-adbeb500-ee12-11ea-904d-40441ebcb32d.png)

**Additional context**
I assume that because the software does not believe that I have files that are labeled, it is unable to create a training data-set. It did say output a FileNotFoundError and looked at a different sub file directory for a file that did not exist in the folder.

**Comment by MMathisLab** [comment]
are you opening the terminal with "as an admin?" it might be nothing can be saved as you don't have permission to do so...

**Comment by WrathUM** [comment]
I am running terminal regularly. Do I need to run it as admin for things to be able to function properly? I am trying to work on the lab computer virtually through TeamViewer on a "non-administrator" account. Does this create additional complications?

**Comment by MMathisLab** [comment]
^ yes, unfortunately this can cause issues, But if it's TeamViewer, just right click on the terminal icon, and select "open as admin"

**Comment by WrathUM** [comment]
![image](https://user-images.githubusercontent.com/57931859/92821880-62f7ec80-f380-11ea-830c-3597152813d7.png)
I tried changing the directory to match the directory that I would be in when using conda activate DLC-GPU. It doesn't recognize the command.

**Comment by AlexEMG** [comment]
You need to add the conda path to the system first — see here https://stackoverflow.com/questions/54501167/anaconda-and-git-bash-in-windows-conda-command-not-found

Sent from my iPhone
